modules/odc-accessing-developer-perspective.adoc

```diff
@@ -10,6 +10,9 @@ The *Developer* perspective in the {product-title} web console provides workflow
 
 You can access the *Developer* perspective from the web console as follows:
 
+.Prerequisites
+To access the *Developer* perspective, ensure that you have logged in to the web console.
+
 .Procedure
 
 . Log in to the {product-title} web console using your login credentials. The default view for the {product-title} web console is the *Administrator* perspective.
```

Comment:
```
The default view for the {product-title} web console is the *Administrator* perspective
```
for administrator the default view is `Administrator` perspective while for non-admin users, the default view is `Developer` perspective